helm_deploy/values-dev.yaml

```diff
@@ -40,3 +40,5 @@ env:
   HMPPS_COOKIE_NAME: hmpps-session-dev
   REMOTE_AUTH_STRATEGY: "true"
   WEB_SESSION_TIMEOUT_IN_MINUTES: "60"
+  TOKENVERIFICATION_API_URL: "https://token-verification-api-dev.prison.service.justice.gov.uk"
+  TOKENVERIFICATION_API_ENABLED: true
```

Comment:
Should this be "true"?

Reply:
when we added it to manage-hmpps-auth-users we enabled it for dev when the health check went in but will check before merge

Reply:
yes, no need for it in quotes